.github/workflows/create-production-agent.yaml

```diff
@@ -0,0 +1,83 @@
+name: create-production-agent
+
+on:
+  workflow_dispatch:
+    inputs:
+      agent_name:
+        description: "Agent name to create (e.g., vargas-jr for production)"
+        required: false
+        type: string
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+      - name: Cache Playwright browsers
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/ms-playwright
+          key: playwright-browsers-${{ runner.os }}-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            playwright-browsers-${{ runner.os }}-
+      - name: Install Node.js dependencies
+        run: npm install
+      - name: Run ESLint
+        run: npm run lint
+      - name: Run Prettier check
+        run: npm run format:check
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+      - name: Cache Playwright browsers
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/ms-playwright
+          key: playwright-browsers-${{ runner.os }}-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            playwright-browsers-${{ runner.os }}-
+      - name: Install Node.js dependencies
+        run: npm install
+      - name: Run tests
+        run: npm test tests/
+  create-production-agent:
+    runs-on: ubuntu-latest
+    needs: [lint, test]
```

Comment:
Remove this dependency and remove the steps. We can assume that we have a valid artifact at this point